.github/workflows/scripts/generate-weekly-report.js

```diff
@@ -247,6 +268,11 @@ function generateReport({ issuesData, previousReport, componentData }) {
     section.push('</li>');
     out.push(section.join('\n'));
   }
+
+  let {count, section} = generateComponentsLookingForOwnersReportSection(lookingForOwners)
+  out.push(`<li> looking for owners: ${count}`)
```

Comment:
```suggestion
  out.push(`<li> Seeking new code owners: ${count}`)
```
Nit: This would make terminology consistent with the README output.